tests/unit/test_calculatePhase0.py

```diff
@@ -15,7 +15,7 @@ def test_calculate(self):
 
         nw = DEFAULT_NETWORK_CONFIG_MAP[CURRENT_TESTNET]
 
-        api = ProviderFactory(provider='prpc').newRewardApi(nw, BAKING_ADDRESS, '')
+        api = ProviderFactory(provider='tzkt').newRewardApi(nw, BAKING_ADDRESS, '')
```

Comment:
Because https://granadanet.smartpy.io return 502 I use tzkt api.

Reply:
I updated the the testnet URL